hack/build/run-lint-checks.sh

```diff
@@ -20,11 +20,25 @@
 source hack/build/config.sh
 source hack/build/common.sh
 
+# Temporary var for pkgs that are ready to enforce linting
+LINTABLE=(pkg/controller pkg/apis pkg/client pkg/controller pkg/expectations pkg/image pkg/importer pkg/lib pkg/util)
+
 ec=0
 out="$(gofmt -l -s ${SOURCE_DIRS} | grep ".*\.go")"
 if [[ ${out} ]]; then
     echo "FAIL: Format errors found in the following files:"
     echo "${out}"
     ec=1
 fi
+
+for p in "${LINTABLE}"
+do
+  out="$(golint ${p}/...)"
+    if [[ ${out}  ]]; then
+    echo "FAIL: following golint errors found:"
```

Comment:
Indent here and below. Does `shfmt` report this?

Reply:
Ahh... yeah; and more importantly that for call to iterate on the array is wrong (missing the extra [@])